src/Content/Backend/Solution/Monaco.Template.Backend.Domain.Tests/GpsPositionTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using FluentAssertions;
+using Monaco.Template.Backend.Domain.Model;
+using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography;
+using Xunit;
+
+namespace Monaco.Template.Backend.Domain.Tests;
+
+[ExcludeFromCodeCoverage]
+[Trait("Core Domain Entities", "GpsPosition Entity")]
+public class GpsPositionTests
+{
+	[Fact(DisplayName = "New GpsPosition succeeds")]
+	public void NewGpsPositionSucceeds()
```

Comment:
Should also test each of the failing cases?

Latitude: -91
Latitude: 91
Longitude: -181
Longitude: 181

Reply:
Indeed, I'm gonna rewrite the params to include the 3 scenarios.

Reply:
I tested locally with this if it helps:

```csharp
	[InlineData(-91, 0)]
	[InlineData(91, 0)]
	[InlineData(0, -181)]
	[InlineData(0, 181)]
	[Theory(DisplayName = "New GpsPosition with invalid positions fails")]
	public void NewGpsPositionWithInvalidPositionsFails(int latitude, int longitude)
	{
		var sut = () => new GpsPosition(latitude, longitude);
		sut.Should().Throw<ArgumentOutOfRangeException>();
	}
```

Reply:
Yup, exactly much the same.